modules/openapi-generator/src/main/resources/swift5/modelEnum.mustache

```diff
@@ -1,7 +1,32 @@
-{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable {
+{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable{{^generateFrozenEnums}}{{#isInteger}}, CaseIterableDefaultsLast{{/isInteger}}{{#isFloat}}, CaseIterableDefaultsLast{{/isFloat}}{{#isDouble}}, CaseIterableDefaultsLast{{/isDouble}}{{#isString}}, CaseIterableDefaultsLast{{/isString}}{{/generateFrozenEnums}} {
 {{#allowableValues}}
 {{#enumVars}}
     case {{{name}}} = {{{value}}}
 {{/enumVars}}
 {{/allowableValues}}
+{{^generateFrozenEnums}}
+    // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
+    // If new enum cases are added that are unknown to the spec/client, they are safely
+    // decoded to this case. The raw value of this case is a dummy value that attempts
+    // to avoids collisions with previously specified cases.
+{{#isString}}
+    case unknownDefaultOpenApi = "unknown_default_open_api"
```

Comment:
@jarrodparkes this is my fault, and sorry for that, but it should be `unknownDefaultOpenAPI`, of OpenAPI